server/core/src/access_time_queue.cpp

```diff
@@ -34,6 +37,14 @@ namespace irods::access_time_queue
             return;
         }
 
+        if (_queue_name_prefix.empty()) {
+            THROW(CONFIGURATION_ERROR, fmt::format("{}: Access time queue name is empty.", __func__));
+        }
+
+        if (constexpr std::size_t max = 500'000; _queue_size > max) {
+            THROW(CONFIGURATION_ERROR, fmt::format("{}: Access time queue size is greater than 500000.", __func__));
```

Comment:
make sure that goes in the docs

Reply:
Definitely.

500k updates is a lot of updates. I don't think anyone's deployment would ever hit that since the server is checking the queue frequently, but we'll see.

NTS: Use `max` in the format string instead of the hard-coded 500000. And update the documentation in the header file to state the max queue size.

Reply:
Updated format string to use `max` variable.
Updated header to mention queue size constraint.

Leaving unresolved to serve as a reminder to document this in irods/irods_docs.

Reply:
Created #8312 and linked it to this conversation.